docs: add ttl control section and faq (#20)
bisdn/bisdn.github.io: setup/setup_standalone.md

Note the absence of client_drivshell and the single exit statement at the end.

### TTL controls

BISDN Linux has a default configuration file under `/etc/ofdpa/rc.soc` controlling both the Broadcom SDK and OF-DPA.

The file has the following control by default,

```
SwitchControl L3UcastTtl1ToCpu=1
```

To enable routing packets with TTL=1 to CPU. 
The following command

```
client_drivshell SwitchControl L3UcastTtl1ToCpu=1
```

> Will activate the same behaviour on the switch in a non-persistent way. By omitting the assignment the status is reported.

Reviewer: ```suggestion
Will activate the same behaviour on the switch in a non-persistent way.
To print the currently set value, you can run:

```
client_drivshell SwitchControl L3UcastTtl1ToCpu
```
```

Note the absence of client_drivshell and the single exit statement at the end.

### TTL controls

BISDN Linux has a default configuration file under `/etc/ofdpa/rc.soc` controlling both the Broadcom SDK and OF-DPA.

The file has the following control by default,

```
SwitchControl L3UcastTtl1ToCpu=1
```

To enable routing packets with TTL=1 to CPU. 
The following command

```
client_drivshell SwitchControl L3UcastTtl1ToCpu=1
```

Will activate the same behaviour on the switch in a non-persistent way.
To print the currently set value, you can run:

## Bundled software with BISDN Linux

> 

Author: ```suggestion

client_drivshell SwitchControl L3UcastTtl1ToCpu=1
``` 
```

Note the absence of client_drivshell and the single exit statement at the end.

### TTL controls

BISDN Linux has a default configuration file under `/etc/ofdpa/rc.soc` controlling both the Broadcom SDK and OF-DPA.

> The file has the following control by default,

Reviewer: ```suggestion
To enable routing packets with TTL=1 to CPU by default, the file contains the following setting:
```

```
SwitchControl L3UcastTtl1ToCpu=1
```

> To enable routing packets with TTL=1 to CPU. 

Reviewer: ```suggestion
```

> The following command

Reviewer: ```suggestion
The following command will activate the same behaviour on the switch in a non-persistent way:
```

```
client_drivshell SwitchControl L3UcastTtl1ToCpu=1
```

> Will activate the same behaviour on the switch in a non-persistent way.

Reviewer: ```suggestion
```
